meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,119 @@
+Items that you want to sell on the Wayfair market must be considered active in Plentymarkets. The Plentymarkets user may also choose limit which Items are for sale on Wayfair. **Note that Inventory and ordered items are controlled at the `Variation` level.**
+
+To ensure that an Item is available for sale, follow these instructions:
+
+1. From the main Plentymarkets page, go to `Item` >> `Edit item`
+
+2. Search for item(s) and open them
+
+3. **For each item**, click `Variations` and open them
+
+4. **For each Variation**:
+
+    1. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability Section`is checked.
+
+    2. If [the `Send all inventory items to Wayfair?` setting](settings_guide.md#send-all-inventory-items-to-wayfair) is **disabled**, go to the `Availability` tab of the Variation and add "Wayfair" to the list in the `Markets` area.
+
+    3. Click the `Save` button next to the Variation `ID` (not the higher-up button for the Item).
+
+
+## 7. Configuring the Warehouse mappings to match Wayfair Supplier IDs.
+
+In order to update the inventory data in Wayfair's system, you need to map the Warehouses in your Plentymarkets system to the Supplier IDs in Wayfair's system, on the [Warehouses](settings_guide.md#warehouses-page) page of the plugin's settings.
+
+## 8. Configuring Plentymarkets to send Confirmation of Delivery (ASN) to Wayfair
+
+### 8.1 Setting the Wayfair Plugin to send the correct shipping information to Wayfair
+Wayfair Plugin users that wish to ship orders by using their own accounts (rather than using Wayfair's shipping services) must update the [Ship Confirmation (ASN) configuration settings](https://github.com/wayfair-contribs/plentymarkets-plugin/blob/master/meta/documents/user_guide/en/settings_guide.md#ship-confirmation-asn-page) to reflect their specific configuration.
+
+If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings should be left in their default (`Wayfair Shipping`) state.
+
+### 8.2 Creating an Event for Plentymarkets Orders that sends shipment information to Wayfair
+
+1. Click **Setup** in the top navigation bar then go to **Orders >> Events**
+![Create Event](https://i.ibb.co/NjDtY05/asn-02.png "Create Event")
+
+2.	Click on **Add event procedure** (the "+" button on the left of the page)
+
+3.	Enter any **Name**
+
+4.	Select **Status change** (in the category **Order Change**) in the **Event** field.
+
+5.	In the field below **Event** select the status change that should initiate the sending of an ASN to Wayfair, such as **in preparation for shipping**.
+
+6.	Click the **Save** button.
+
+7.	You should automatically be redirected to the newly created **event procedure**. In the **settings** section of the event procedure, place a checkmark next to **Active**.
+
+8.  Click on **Add Filter**, and go to **Order >> Referrer** to add the referrer as a filter (see screenshot below)
+
+![Event Referrer](https://i.ibb.co/TwKLvJ5/asn-03.png "Event Referrer")

```

Comment:
I may be mis-remembering, but I thought you mentioned soething about hosting the images on github instead of some image hosting service (this comment also applies to Lines 106, 110, 114)

Reply:
You are correct. We'll fix that in a future update. see the notices in the PR description.